django/core/management/commands/makemigrations.py

```diff
@@ -75,6 +76,11 @@ def handle(self, *app_labels, **options):
         # the loader doesn't try to resolve replaced migrations from DB.
         loader = MigrationLoader(None, ignore_no_migrations=True)
 
+        # For every connection check if all migartions are applied after their
```

Comment:
I think it would be more clear to write "check that there aren't any migrations applied before their dependencies."